src/MOIWrapper.jl

```diff
@@ -0,0 +1,349 @@
+    end
+    cbcModelFormat.row_lb[mapping.conmap[ci].value] = s.lower - f.constant
+end
+
+function loadConstraint(ci::MOI.ConstraintIndex, cbcModelFormat::CbcModelFormat, mapping::MOIU.IndexMap,
+    f::MOI.ScalarAffineFunction, s::MOI.LessThan)
+    for term in f.terms
+        cbcModelFormat.constraint_matrix[mapping.conmap[ci].value,
+        mapping.varmap[term.variable_index].value] += term.coefficient
+    end
+    cbcModelFormat.row_ub[mapping.conmap[ci].value] = s.upper - f.constant
+end
+
+function loadConstraint(ci::MOI.ConstraintIndex, cbcModelFormat::CbcModelFormat, mapping::MOIU.IndexMap,
+    f::MOI.ScalarAffineFunction, s::MOI.Interval)
+    for term in f.terms
+        cbcModelFormat.constraint_matrix[mapping.conmap[ci].value,
+        mapping.varmap[term.variable_index].value] += term.coefficient
+    end
+    cbcModelFormat.row_ub[mapping.conmap[ci].value] = s.upper - f.constant
+    cbcModelFormat.row_lb[mapping.conmap[ci].value] = s.lower - f.constant
+end
+
+
+function loadObj(cbcModelFormat::CbcModelFormat, mapping::MOIU.IndexMap,
+    f::MOI.ScalarAffineFunction)
+    for term in f.terms
+        cbcModelFormat.obj[mapping.varmap[term.variable_index].value] += term.coefficient
+    end
+end
+
+function copyconstraints!(cbcModelFormat::CbcModelFormat, userOptimizer::MOI.ModelLike,
+    mapping::MOIU.IndexMap)
+    for (F,S) in MOI.get(userOptimizer, MOI.ListOfConstraints())
+        if !(S <: Union{MOI.ZeroOne, MOI.Integer})
+            for ci in MOI.get(userOptimizer, MOI.ListOfConstraintIndices{F,S}())
+                f = MOI.get(userOptimizer, MOI.ConstraintFunction(), ci)
+                s = MOI.get(userOptimizer,  MOI.ConstraintSet(), ci)
+                loadConstraint(ci, cbcModelFormat, mapping, f, s)
+            end
+        end
+    end
+end
+
+function updateBoundsForBinaryVars!(col_lb::Vector{Float64}, col_ub::Vector{Float64}, zeroOneIndices::Vector{Int})
+    for colIdx in zeroOneIndices
+        if col_lb[colIdx] < 0.0
+            col_lb[colIdx] = 0.0
+        end
+        if col_ub[colIdx] > 1.0
+            col_ub[colIdx] = 1.0
+        end
+    end
+end
+
+function updateZeroOneIndices(userOptimizer::MOI.ModelLike, mapping::MOIU.IndexMap,
+    ci::Vector{MOI.ConstraintIndex{F,S}}, zeroOneIndices::Vector{Int}) where {F, S}
+    for i in 1:length(ci)
+        f = MOI.get(userOptimizer, MOI.ConstraintFunction(), ci[i])
+        push!(zeroOneIndices, mapping.varmap[f.variable].value)
+    end
+end
+
+function updateIntegerIndices(userOptimizer::MOI.ModelLike, mapping::MOIU.IndexMap,
+    ci::Vector{MOI.ConstraintIndex{F,S}}, integerIndices::Vector{Int}) where {F, S}
+    for i in 1:length(ci)
+        f = MOI.get(userOptimizer, MOI.ConstraintFunction(), ci[i])
+        push!(integerIndices, mapping.varmap[f.variable].value)
+    end
+end
+
+"""
+    Create inner model cbcOptimizer based on abstract model userOptimizer provided by user.
```

Comment:
If you indent, in markdown, it means that this is code.
You should rather put an indented signature followed by an unindented description :)